scripts/ansible/roles/ava-install/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+ava_daemon_bin_dir: "/usr/bin"
```

Comment:
I personally would prefer `/usr/local/bin` - this will be the default installation path for binary releases.

Reply:
Fair point, I'll change it.